lua/luasnip/extras/postfix.lua

```diff
@@ -0,0 +1,75 @@
+local snip = require("luasnip").snippet
+local events = require("luasnip.util.events")
+local matches = {
+	default = [[[%w%.%_%-%"%']+$]],
+	line = "^.+$",
```

Comment:
Does this have any use currently?

Reply:
no, but I was thinking this section could be a set of commonly used matches, for people to plug in to the postfix snippets. We could totally get rid of it though

Reply:
Ahh, I didn't see that it was exported.
Let's keep it, it should be useful, would you mention that in the doc as well?